### PR TITLE
[credsweeper] Initial commit

### PR DESCRIPTION
**Pull request by babenek**
Create initial config file with required info, before configurations

Credential leakage from repo source code is a huge vulnerability that might happen to any developer at any moment 
Modern tools for credential detection can automatically detect if developer accidentally committed their cred (password, key, etc). However, their disadvantage is the following: Too many false reports. Developers just don’t want to use them, as amount of false reporting is too high

That is why SR presented CredSweeper during Samsung Security Tech forum 2021
https://youtu.be/RDl81Jd83zc?t=14717 

This scanner allows detecting credentials with much lower false positive rate than existing alternatives 

SR opensourced both tool and related credential dataset, so other people can contribute and improve credential detection tools, or just use our tool to protect their repos from accidental credential leakage

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
Could you write a bit about why credsweeper is a critical open source project, i.e. who are the customers of credsweeper?

**Comment by babenek** [comment]
> Could you write a bit about why credsweeper is a critical open source project, i.e. who are the customers of credsweeper?

As i know exactly - several projects of Samsung use the project. Suppose, Hyundai projects too https://github.com/Samsung/CredSweeper/network/members

File: projects/credsweeper/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-python
+RUN git clone \
+    --depth 1 \
+    --branch oss-fuzz \
+    https://github.com/babenek/CredSweeper.git
```
Comment (babenek) [inline projects/credsweeper/Dockerfile:21]: @DavidKorczynski DO NOT MERGE until PR use debug fork

**Comment by DavidKorczynski** [comment]
> @DavidKorczynski, the PR is ready. CredSweeper main repo is prepared to fuzzing without tensorflow. It takes much time for instrumenting and does not real coverage.

I reran the CI  but it's still failing

**Comment by DavidKorczynski** [comment]
> > Could you write a bit about why credsweeper is a critical open source project, i.e. who are the customers of credsweeper?
> 
> As i know exactly - several projects of Samsung use the project. Suppose, Hyundai projects too https://github.com/Samsung/CredSweeper/network/members

@jonathanmetzman @oliverchang this one is ready for acceptance review. The first message of the PR also has notes on the project.

**Comment by babenek** [comment]
@DavidKorczynski thanks, i'm investigating what's wrong...

**Comment by jonathanmetzman** [comment]
One mistake is you need a license header in /home/runner/work/oss-fuzz/oss-fuzz/projects/credsweeper/run.sh.

**Comment by jonathanmetzman** [comment]
Let me ask the panel to consider accepting this.

**Comment by babenek** [comment]
well, if possibly to postpone final configuration...

**Comment by babenek** [comment]
Dears @jonathanmetzman @oliverchang , review please the PR. I think, all checks will pass.

**Comment by DavidKorczynski** [comment]
> Dears @jonathanmetzman @oliverchang , review please the PR. I think, all checks will pass.

triggered CI run

**Comment by babenek** [comment]
All tests passed :)

**Comment by babenek** [comment]
@DavidKorczynski @jonathanmetzman , could you give a decision about PR or estimate PR deadline? Thanks.

**Comment by babenek** [comment]
Rebased to latest master to skip failures in other projects

**Comment by babenek** [comment]
forked build https://github.com/babenek/oss-fuzz/pull/3/checks

**Comment by jonathanmetzman** [comment]
Hi @babenek 
We don't really think credsweeper has the userbase we want for acceptance into OSS-Fuzz yet.
Would you like help in setting up [clusterfuzzlite](https://google.github.io/clusterfuzzlite). Since ClusterFuzzLite is a new project I'm very happy to help potential users.

**Comment by jonathanmetzman** [comment]
Send me an email metzman@chromium.org if you want help with ClusterFuzzLite

**Pull request closed without merging**